airflow/contrib/operators/gcp_sql_operator.py

```diff
@@ -286,3 +289,172 @@ def execute(self, context):
+    by postgres and mysql.
+
+    :param sql: SQL query or list of queries to run (should be DML or DDL query -
+        this operator does not return any data from the database,
+        so it is useless to pass it DQL queru.
+    :type sql: str or [str]
+    :param project_id: Google Cloud Platform project where the Cloud SQL instance exists.
+    :type project_id: str
+    :param location: Location of the Cloud SQL database instance (for example
+         'europe-west1').
+    :type location: str
+    :param instance: Name of the instance of the Cloud SQL database instance.
+    :type instance: str
+    :param database_type: Type of the database instance (mysql or postgres)
+    :type database_type: str
+    :param use_proxy: Whether SQL proxy should be used to connect to Cloud SQL DB
+    :type use_proxy: bool
+    :param sql_proxy_use_tcp: If set to true, TCP is used to connect via proxy.
+       By default UNIX sockets are used). Requires sql_proxy_tcp_port to be set.
+    :type sql_proxy_use_tcp: bool
+    :param sql_proxy_tcp_port: port number to use by cloud-sql-proxy
+    :type sql_proxy_tcp_port: int
+    :param sql_proxy_version: specific version of the proxy to download. If not specified
+       the latest version is downloaded.
+    :type sql_proxy_version: str
+    :param sql_proxy_force_download: Whether to force download of sql proxy every time
+    :type sql_proxy_force_download: bool
+    :param cloudsql_conn_id: reference to a specific connection id defined.
+        Note that in case use_proxy is set to true, this connection should be properly
+        specified - so that they are directed at the proxy. It's host should be
+        "/tmp/gcsp_sockets_<task_id>/<project_id>:<location>:<instance>"
+        in case of UNIX sockets or "127.0.0.1:<port>" in case of TCP connection.
+        Also in case use_proxy is set to True, the connections should be non-SSL
+        (cloud-sql-proxy provides encrypted tunnel to the Cloud SQL database).
+        In real environment the connections might also be specified via UI,
+        but you can also specify them via OS environment variables as
+        described in https://airflow.apache.org/howto/manage-connections.html.
+        Length of the task id is limited due to socket path limitation:
+        http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_un.h.html#tag_13_67_04
+    :type cloudsql_conn_id: str
+    :param parameters: (optional) the parameters to render the SQL query with.
+    :type parameters: mapping or iterable
+    :param autocommit: if True, each command is automatically committed.
+        (default value: False)
+    :type autocommit: bool
+    :param database: name of database which overwrites the name defined in the connection
+    :type database: str
+    :param gcp_conn_id: The connection ID used to connect to Google Cloud Platform for
+       cloud-sql-proxy authentication.
+    :type gcp_conn_id: str
```

Comment:
Again, inconsistencies as to lower- / upper- case first letter and ending with a `.` or not.